sdk/go/.changes/unreleased/Fixed-20230720-184344.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Added
+body: print to stdout when a new Engine session & connection are establishing
```

Comment:
```suggestion
body: Print progress status during automatic Engine provisioning and establishing client connection
```

Reply:
Done in https://github.com/dagger/dagger/pull/5488/commits/39184eac851ad84992a63c2d5d4caff5deed22b3